test_config.yaml

```diff
@@ -40,3 +40,23 @@ notifications:
   enabled: True
 
 testing: true
+
+services:
+  paths:
+    - ./baselayer/services
+    - ./skyportal/services
+
+test_server:
+  port: 64502
+  # Reset the cache every * days
+  refresh_cache_days: 7
```

Comment:
This refresh rate will be changeable for each system? We won't want to have to test LCO too often given the program restrictions, for example.

Reply:
That's a good point that I hadn't thought of. As I have it now, requests for everything (LT, LCO, etc) will all go into the same cache file, and the refresh rate is defined for the entire file. Unfortunately VCR doesn't really have good support for request-specific expiration dates as far as I know, so I think we probably have to do something like a separate cache file for each system in order to support different refresh rates for each one.

Reply:
Ok makes sense!